salt/docker/remove-pre-reboot.sls

```diff
@@ -0,0 +1,18 @@
+# cleanup all the things we have created
+
+/etc/systemd/system/docker.service.d/proxy.conf:
+  file.absent
+
+/etc/docker/daemon.json:
+  file.absent
+
+/etc/sysconfig/docker:
+  file.absent
+
+docker-wipe-etc-certs:
+  cmd.run:
+    - name: rm -rf /etc/docker/certs.d/*
```

Comment:
`file.absent` can also be used here, it should also recursively remove directories.

Reply:
Yes, I know about it, problem with `file.absent`  is that it can't remove content of directory. It will just wipe whole directory. I think we would like to leave empty path like it was before installation. But I'm open for using `file.absent` as well

Reply:
I think it's not a problem since we don't plan to reuse the node, it needs to be reinstalled.

Reply:
ok, then I will change it